cli/azd/pkg/input/console.go

```diff
@@ -441,6 +444,16 @@ func (c *AskerConsole) Confirm(ctx context.Context, options ConsoleOptions) (boo
 	return response, nil
 }
 
+// wait until the next enter
+func (c *AskerConsole) WaitForEnter() {
+	inputScanner := bufio.NewScanner(c.handles.Stdin)
+	if scan := inputScanner.Scan(); !scan {
+		if err := inputScanner.Err(); err != nil {
+			log.Printf("error while waiting for enter: %v", err)
```

Comment:
Would you consider just returning `err` here, or `panic`? It seems that continuing here may be unexpected if an unexpected error occurred while reading `Stdin`

Reply:
I would like to wait until that happen to make a decision.  Having a panic might be as bad as having unexpected behavior... Since I can't think about how and what the error might be here (an error trying to read stdin might be a bigger error with the system and not really azd), I would rather wait

Reply:
My preference is that when any system acts abnormally, I think the user and the developer wants to know instead of things pretending as-if they are normal.

Reply:
> I think the user and the developer wants to know instead of the program pretending as-if they are normal.

Is this possible? 

The `user` and the `developer` live in a different moment of the application lifetime.
Developers use `panic` to be addressed while running test cases.  If you ask me, a user looking at a `panic` event is developer's scape. It is an error in the code which should be fix by developer.